android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsModule.java

```diff
@@ -120,7 +120,8 @@ private RequestConfiguration buildRequestConfiguration(ReadableMap requestConfig
   @ReactMethod
   public void initialize(Promise promise) {
     MobileAds.initialize(
-        getApplicationContext(),
+        /* Use activity when available - preferrable for mediated networks*/
+        (getCurrentActivity() != null) ? getCurrentActivity() : getApplicationContext(),
```

Comment:
Is the second call to getCurrentActivity guaranteed the return the same result as the first one?